README.md

```diff
@@ -11,66 +11,122 @@ pages with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/).
 
 ## How to do a Quick Edit on github
 
-To do a quick edit, use this is a 3 step process:
+To do a quick edit, use this is a 4 step process:
 
-1st: Make your edit and commit to the develop branch
+1st: Make your edit and commit to a new branch.
 
-2nd: Make a PR from develop to master, get that reviewed
+2nd: Make a PR from your branch to develop, get that reviewed.
 
-3rd: Merge the PR to master
+3rd: Merge the PR to develop.
 
-This commits directly to the `develop` branch. If you are doing a more involved
-group of edits or changes, please use a process like git flow. If you are not
-familiar with this ask a dev or an admin to help you.
+4th: Create a PR to deploy your changes to master.
```

Comment:
I would say that this change is not really a necessary step for the audience. The merging from `develop` to `master` should be decided independently.

Reply:
Agreed. The whole point of `develop` branch is to stop and review it on info.dev.arxiv.org, not just a train stop you have to run through on the way to `master`

Reply:
Yeah, let's not make this step 4, but rather something that happens only after the preview on  info.dev.arxiv.org

Reply:
Okay. Happy to remove. If we don't want it as a step 4. Do we want provide any guidance to editors on what to do after their content is approved on develop? 

Should they create a ticket and ask someone to create a PR and merge it for them?